test/utils/generate-py.sh

```diff
@@ -1,7 +1,7 @@
 # Copy this file to compiler test folder where the .npy files are generated.
 # Update SIMULATOR_PATH value and execute it.
 
-SIMULATOR_PATH=" " # simulator root path
+SIMULATOR_PATH="/home/marte/Desktop/git_fork/puma-simulator" # simulator root path
```

Comment:
Do not commit personal paths.

```suggestion
SIMULATOR_PATH=" " # simulator root path
```